.gitignore

```diff
@@ -39,6 +39,7 @@
 *~
 *$
 *.bak
+.idea
```

Comment:
This is for PyCharm and would be nice to have.

Reply:
Personally I don't like this, but other core devs don't mind (we had multiple discussions/arguments about this subject before) and we already have a long list of other editors' files included, so it's fine. I would, however, suggest configuring global `~/.gitignore` and/or per repository `./.git/info/exclude` (see more [here](https://git-scm.com/docs/gitignore)) if working with multiple repositories, especially ones you don't own.